docker/init-db.sql

```diff
@@ -336,4 +453,113 @@ CREATE TABLE model_deployment (
+                       created_at TIMESTAMP,
+                       created_by INTEGER REFERENCES personnel(person_id) ON DELETE CASCADE,
+                       last_updated_at TIMESTAMP,
+                       last_updated_by INTEGER REFERENCES personnel(person_id) ON DELETE CASCADE
+);
+
+-- Insert dummy model
+INSERT INTO model (model_id, learning_process_id, study_id, name, version, tag, model_type, product_identifier,
+                   owner, trl_level, license, primary_use, secondary_use, intended_users, counter_indications,
+                   ethical_considerations, limitations, fairness_constraints, created_at, created_by,
+                   last_updated_at, last_updated_by) VALUES
+    (1, 1, 1, 'test_name', 'test_version', 'test_tag', 'test_model_type', 'test_product_identifier', 1,
+     'test_trl_level', 'test_license', 'test_primary_use', 'test_secondary_use', 'test_intended_users',
+     'test_counter_indications', 'test_ethical_considerations', 'test_limitations', 'test_fariness_constraints',
+     '2023-01-01 00:00:00', 1, '2023-01-02 00:00:00', 1);
+
+
+
+-- Create model_deployment table
+CREATE TABLE model_deployment (
+                                  deployment_id SERIAL PRIMARY KEY,
+                                  model_id INTEGER REFERENCES model(model_id),
+                                  environment_id INTEGER REFERENCES deployment_environment(environment_id),
+                                  tags VARCHAR(255),
+                                  identified_failures TEXT,
+                                  status VARCHAR(255),
+                                  created_at TIMESTAMP,
+                                  created_by INTEGER REFERENCES personnel(person_id),
+                                  last_updated_at TIMESTAMP,
+                                  last_updated_by INTEGER REFERENCES personnel(person_id)
+);
+
+-- Insert dummy model_deployment
+INSERT INTO model_deployment (deployment_id, model_id, environment_id, tags, identified_failures, status, created_at, created_by, last_updated_at, last_updated_by) VALUES
+    (1, 1, 1, 'Production', 'Instances of false positives in predicting rare events.', 'RUNNING', '2023-01-01 00:00:00', 1, '2023-01-01 00:00:00', 1);
+
+
+-- Create passport table
+CREATE TABLE passport (
+                                  passport_id SERIAL PRIMARY KEY,
+                                  deployment_id INTEGER REFERENCES model_deployment(deployment_id) ON DELETE CASCADE,
+                                  created_at TIMESTAMP,
+                                  created_by INTEGER REFERENCES personnel(person_id) ON DELETE CASCADE,
+                                  approved_at TIMESTAMP,
+                                  approved_by INTEGER REFERENCES personnel(person_id) ON DELETE CASCADE
+);
+
+-- Insert dummy passport
+INSERT INTO passport (passport_id, deployment_id, created_at, created_by, approved_at, approved_by) VALUES
```

Comment:
Change the Passport - Personnel foreign key connections to VARCHAR(255), and replace the person_id examples with 'service-account-admin'. 

Change the related requests in Postman with respect o these changes as well.